scraping/scraping/spiders/zalando_fr.py

```diff
@@ -0,0 +1,10 @@
+from logging import getLogger
+
+from .zalando import ZalandoSpider
+
+logger = getLogger(__name__)
+
+
+class ZalandoFrSpider(ZalandoSpider):
```

Comment:
I recognized that using `ZalandoSpider` as parent makes almost all lines of code obsolete since they are than inherited.